ingest/src/test/java/com/microsoft/azure/kusto/ingest/E2ETest.java

```diff
@@ -704,22 +716,4 @@ void testStreamingIngestFromBlob() throws IngestionClientException, IngestionSer
             }
         }
     }
-
-    @Test
-    void testProxyPlanner() throws URISyntaxException {
```

Comment:
The routeplanner is an Apache specific concept.

No other HTTP Client has a route planner, including the default Netty HTTP client and OkHttp, as it's not really feasible to plan routes beyond indicating a proxy. It is not supportable outside of the Apache client and it was not included in Apache HttpClient5 either. It was last available in Apache HttpClient4. My recommendation would be to abandon it.

The ProxyOptions in azure http client should provide significant proxy configuration capabilities.

Reply:
We kind of test here the proxy capabilities of the http client - please use the proxy options to achive same results

Reply:
@ohadbitt What are you running on port 8080 of your localhost to forward connections to your test cluster? I'll need to set my system up similarly so that it forwards connections to my test ADX cluster. Python HTTP server or something?